static/bots/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Bots
+---
+
+Hello world 3
```

Comment:
Hello world 3?

Reply:
I was testing the dynamic reload... removed